[receiver/sqlquery] Flaky test TestPostgresIntegrationLogsTrackingWithStorage

### What happened?

Seen in https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/5316701845/jobs/9626920066?pr=23564

```
--- FAIL: TestPostgresIntegrationLogsTrackingWithStorage (6.55s)
    integration_test.go:159: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/sqlqueryreceiver/integration_test.go:159
        	Error:      	Condition never satisfied
        	Test:       	TestPostgresIntegrationLogsTrackingWithStorage
        	Messages:   	failed to receive more than 0 logs
{"level":"error","msg":"error collecting logs","error":"error getting rows: dial tcp [::1]:15431: connect: connection refused","query":"query-0: select * from simple_logs where id > $1"}
```

--- a/receiver/sqlqueryreceiver/integration_test.go
+++ b/receiver/sqlqueryreceiver/integration_test.go
@@ -72,7 +72,7 @@ func TestPostgresIntegrationLogsTrackingWithoutStorage(t *testing.T) {
 		func() bool {
 			return consumer.LogRecordCount() > 0
 		},
-		3*time.Second,
+		1*time.Minute,
 		1*time.Second,
 		"failed to receive more than 0 logs",
 	)
@@ -107,7 +107,7 @@ func TestPostgresIntegrationLogsTrackingWithoutStorage(t *testing.T) {
 		func() bool {
 			return consumer.LogRecordCount() > 0
 		},
-		3*time.Second,
+		1*time.Minute,
 		1*time.Second,
 		"failed to receive more than 0 logs",
 	)
@@ -161,7 +161,7 @@ func TestPostgresIntegrationLogsTrackingWithStorage(t *testing.T) {
 		func() bool {
 			return consumer.LogRecordCount() > 0
 		},
-		3*time.Second,
+		1*time.Minute,
 		1*time.Second,
 		"failed to receive more than 0 logs",
 	)
@@ -195,7 +195,7 @@ func TestPostgresIntegrationLogsTrackingWithStorage(t *testing.T) {
 	require.NoError(t, err)
 
 	// Wait for some logs to come in.
-	time.Sleep(3 * time.Second)
+	time.Sleep(5 * time.Second)
 
 	// stop the SQL Query receiver
 	err = receiver.Shutdown(context.Background())
@@ -233,7 +233,7 @@ func TestPostgresIntegrationLogsTrackingWithStorage(t *testing.T) {
 		func() bool {
 			return consumer.LogRecordCount() > 0
 		},
-		3*time.Second,
+		1*time.Minute,
 		1*time.Second,
 		"failed to receive more than 0 logs",
 	)

